src/explore-education-statistics-admin/src/pages/release/pre-release/PreReleaseContentPage.tsx

```diff
@@ -1,26 +1,57 @@
 import PageTitle from '@admin/components/PageTitle';
+import Link from '@admin/components/Link';
 import ReleaseContent from '@admin/pages/release/content/components/ReleaseContent';
 import { ReleaseContentProvider } from '@admin/pages/release/content/contexts/ReleaseContentContext';
 import { ReleaseRouteParams } from '@admin/routes/releaseRoutes';
-import releaseContentService from '@admin/services/releaseContentService';
+import {
+  preReleaseTableToolRoute,
+  PreReleaseTableToolRouteParams,
+} from '@admin/routes/preReleaseRoutes';
+import featuredTableQueries from '@admin/queries/featuredTableQueries';
+import releaseContentQueries from '@admin/queries/releaseContentQueries';
 import LoadingSpinner from '@common/components/LoadingSpinner';
-import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
 import React from 'react';
 import { RouteComponentProps } from 'react-router';
+import { generatePath } from 'react-router-dom';
+import { useQuery } from '@tanstack/react-query';
 
 const PreReleaseContentPage = ({
   match,
 }: RouteComponentProps<ReleaseRouteParams>) => {
-  const { releaseId } = match.params;
+  const { publicationId, releaseId } = match.params;
 
-  const { value: content, isLoading } = useAsyncHandledRetry(
-    () => releaseContentService.getContent(releaseId, true),
-    [releaseId],
+  const { data: content, isLoading: isLoadingContent } = useQuery(
```

Comment:
Any benefit on parallelising content and featured table loads with `Promise.all`?

Reply:
I don't think we need to now it uses react-query which will run them in parallel.

Reply:
Oh cool!  Didn't realise that